### PR TITLE
Requirements file.

### PR DESCRIPTION
**Pull request by gabrielortegaproyectos**
En esta pull request se agrego los requerimientos del entrono virtual.

### CODE REVIEW

File: README.md
Comment (framunoz) [inline README.md:1]: Why are you including this stuff??!!11

File: README.md [hunk 1]
```diff
@@ -1,2 +1,61 @@
...
+
+
+
+```mermaid
+graph TD
+    A[MisconceptionName] --> B[**MisconceptionType-Model**]
+    B --> D[Modeling Error]
+    B --> E[Calculation Error]
+    B --> F[Interpretation Error]
+```
```
Comment (framunoz) [inline README.md:12]: Why are you including this stuff??!!11

**Comment by framunoz** [comment]
I don't agree

**Review by framunoz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gabrielortegaproyectos) [reply README.md:12]: bad my

Comment (gabrielortegaproyectos) [reply README.md:12]: i will propose a new pull request

Comment (framunoz) [reply README.md:12]: agree

**Pull request closed without merging**